tests/build/multi-platform.go

```diff
@@ -458,6 +458,13 @@ var _ = framework.MultiPlatformBuildSuiteDescribe("Multi Platform Controller E2E
 				}, timeout, interval).Should(Succeed(), "timed out when verifying that the remote host was cleaned up correctly")
 			})
 
+			It("should push Dockerfile to registry", func() {
```

Comment:
Do we need this check here for `multi-platform` test? I think including the check in `build_templates.go` is sufficient.

Reply:
Updated.